### PR TITLE
WIP Front End Refactor

### PR DESCRIPTION
**Pull request by mikefrey**

### CODE REVIEW

File: app/javascript/fog-of-war.ts [hunk 1]
```diff
@@ -22,27 +27,24 @@ export default class FogOfWar {
...
+		super()
+
 		const maskLayer = g.append('g')
 		const mask = svg.append('mask').attr('id', 'mapMask')
 
 		this.maskLayer = maskLayer
 		this.mask = mask
-		this.sections = sections
-		this.channel = channel
+		// this.channel = channel
```
Comment (pbyrne) [inline app/javascript/fog-of-war.ts]: ?

**Comment by pbyrne** [comment]
Feel encouraged to merge this when ready. I'd have done it already, but GitHub saw the "WIP" and disabled the merge button for me without first clicking "ready for review".

File: app/javascript/fog-of-war.ts (hunk 1 above)
Comment (mikefrey) [reply app/javascript/fog-of-war.ts]: There is cleanup to do

**Comment by mikefrey** [comment]
I disabled merge by making it a draft PR

**Pull request merged**